src/SFA.DAS.FAT.Domain/CourseProviders/LocationType.cs

```diff
@@ -2,7 +2,7 @@
 
 public enum LocationType
 {
-    Provider = 0,
-    National = 1,
-    Regional = 2
```

Comment:
dont see a reason to remove this, as I think we sometimes get these values back as 0,1,2? So easier to examine.  I could be wrong though.